app/models/event.rb

```diff
@@ -252,7 +252,7 @@ def issn
   end
 
   def registrant_id
-    [subj["provider_id"], obj["provider_id"]].compact
+    [subj["publisher"], obj["publisher"]].compact
```

Comment:
Why this change? `registrant-id` is intended as an alias for `provider-id`.

Reply:
well the controller is already expecting those are publishers. https://github.com/datacite/lagottino/blob/024c1c035e8ce3d993ef58123017a0e270daeccf/app/controllers/events_controller.rb#L187

I can see levriero to it's sending both:
https://github.com/datacite/levriero/blob/106cb9fbd97ca0b6ad92f2863b8bc2892c7e6552/app/models/base.rb#L220

but the controller only is allowing `publisher`.

I can change the whole thing to one, I just want to hurry up and test-probe that I can definitely use this to make the graphs

Reply:
The `publisher` property is something else in events (see `levriero` code), we need `provider_id`. But you are right that we need `provider_id` in line 187 of the events_controller.

Reply:
plus having the actual name is more useful for the graph. i mean just for my use-case

Reply:
so publisher is not the alphanumeric of provider-id ? mmmmhhh.

Reply:
You need identifiers for the aggregation. Human-readable names is something you add in the `facets` concern. I think it is not a good idea to aggregate using a free-text field.

Reply:
fixed on e09599a